metron-contrib/metron-performance/src/main/java/org/apache/metron/performance/load/SendToKafka.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.metron.performance.load;
+
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.TimerTask;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Future;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.function.Supplier;
+
+public class SendToKafka extends TimerTask {
```

Comment:
well, I mean, I suppose it could be, but I didn't see value in it since all of our inputs currently are strings and the message generator exclusively generates strings, not other messages.  It'd be a more serious refactoring to genericize this to make the message generator pluggable.  Also, for utility code, I think I'd rather avoid prematurely generalizing it.  Let me know what you think.

Reply:
To expand on the reasoning for just making it SendToKafka<K, V>, right now it's using raw KafkaProducer, with one exception.  Since we already know the types, we can make it KafkaProducer<String, String> in the arguments.  I think we shouldn't leave it raw, because  passing in a different KafkaProducer is going to be incorrect the exception at line 71:
`KafkaProducer<String, String> producer = kafkaProducer.get();`.

I definitely don't think making the whole thing pluggable is worth doing now but making the KafkaProducer's all <String, String> for correctness vs just making the class <K, V> is basically the same amount of effort.

Reply:
Ah, so if I'm following you, making the `ThreadLocal<KafkaProducer>` into `ThreadLocal<KafkaProducer<String, String>>` would solve the issue, correct?  I'd prefer to not change to`SendToKafka<K, V>` because it implies that we might be sending something other than String in this implementation.  We would not because the message generator sends Strings and I'd prefer to not generalize that.

Reply:
Yeah, that's a fair point. Yeah, I'd just go ahead with `ThreadLocal<KafkaProducer<String, String>>` and make the arguments to the couple functions that take it `KafkaProducer<String, String>` and I'm good.

Reply:
alright, done and done.